.github/workflows/required.yml

```diff
@@ -13,8 +13,15 @@ jobs:
           - 'fedora_33'
           - 'fedora_latest_jdk11'
           - 'symbolcheck'
+          - 'debian_jdk11'
+          - 'ubuntu_jdk8'
+          # Disable tests due to missing dependencies
+          # - 'centos_7'
+          # - 'centos_8'
```

Comment:
Aww they still haven't gotten an updated cmake version? :'(

Reply:
Yeah, I have no idea what the process is for CentOS 7 & 8. We're working on CentOS 9 though, so we can make sure it's working there.

Reply:
BTW, there seems to be an issue with pkcs11 constants again. I'm trying to investigate it.